site/src/pages/components/invokers.explainer.mdx

```diff
@@ -275,22 +275,24 @@ additional behaviours alongside the event dispatch. In addition the following
 table represents how built-in invocations on specific element types are
 handled. Note that this list is ordered and higher rules take precedence:
 
+When the `invokeaction` attribute is missing it will default to an `auto` state.
+
 | Invokee Element | `action` hint         | Behaviour                                                                            |
 | :-------------- | :-------------------- | :----------------------------------------------------------------------------------- |
-| `<* popover>`   | `'auto'`              | Call `.togglePopover()` on the invokee                                               |
+| `<* popover>`   | `auto`                | Same as `'togglePopover'`                                                            |
```

Comment:
Non blocking but this could be `(auto)` instead of `` `auto` `` for extra clarity, perhaps.
```suggestion
| `<* popover>`   | (auto)                | Same as `'togglePopover'`                                                            |
```

Reply:
Ah good shout